### DISCUSSION TITLE
Lithuanian Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#372] [Language Leaders]
@ErnestStaug - should be the right place now :slightly_smiling_face:

**Comment by ernst77** [comment]
@synesthesiam It is indeed :)

**Comment by synesthesiam** [comment]
@ErnestStaug when you get a chance, could you please look into translating the `HassGetState` sentences/responses?

**Reply by ernst77** [reply]
Need a fix on  #1077 as depending on a sentence we use different word endings for entity name. Looks like matched/unmatched array is not getting populated if there is alternative words/phrases in  entity name.

**Reply by synesthesiam** [reply]
I pushed a PR today for this, can you update and check again?

**Reply by ernst77** [reply]
Yes, seems to work now.

**Comment by blablazzz** [comment]
cant select lithuanian (no such option) in voice assistant configuration menu. O see only latvian and latin under letter L.

**Reply by synesthesiam** [reply]
@blablazzz Can you file a bug for HA core and tag me please? This shouldn't be happening.

**Reply by ernst77** [reply]
@synesthesiam made an issue https://github.com/home-assistant/core/issues/92846
